dashboard/src/index.html

```diff
@@ -8,12 +8,15 @@
 
   <link href="img/favicon.ico" rel="shortcut icon" type="image/vnd.microsoft.icon" />
   <!-- STYLES -->
-  <link rel="stylesheet" type="text/css" href="css/index.css">
   <!-- build:css lib/css/main.min.css -->
   <link rel="stylesheet" type="text/css" href="components/bootstrap/dist/css/bootstrap.min.css">
   <link rel="stylesheet" type="text/css" href="components/font-awesome/css/font-awesome.min.css">
   <link rel="stylesheet" type="text/css" href="components/rdash-ui/dist/css/rdash.min.css">
   <!-- endbuild -->
+
```

Comment:
@mariosant Hey! Nice to meet you. I'm a bit confused as to what you mean. The file I am referring to (index.css) was never concatenated or minified before this commit at all. I just jumped on to help fix a design issue. 

I didn't add a new stylesheet if that's what you're referring to, I used the one already supplied in the structure. All that I changed was the positioning of the "+  <link rel="stylesheet" type="text/css" href="css/index.css">" code from above the other CSS to below it so that the custom CSS file (index.css) takes preference over the files we're pulling externally.

I'm not trying to argue, just understand where we're all at. Can we agree that we will setup index.css to be minified to index.min.css and then update the rest of the HTML in that regard?